examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -1,71 +1,88 @@
 """
-==============================================================
-Plot Ridge coefficients as a function of the L2 regularization
-==============================================================
+=========================================================
+Ridge coefficients as a function of the L2 regularization
+=========================================================
 
 .. currentmodule:: sklearn.linear_model
 
-:class:`Ridge` Regression is the estimator used in this example.
-Each color in the left plot represents one different dimension of the
-coefficient vector, and this is displayed as a function of the
-regularization parameter. The right plot shows how exact the solution
-is. This example illustrates how a well defined solution is
-found by Ridge regression and how regularization affects the
-coefficients and their values. The plot on the right shows how
-the difference of the coefficients from the estimator changes
-as a function of regularization.
-
-In this example the dependent variable Y is set as a function
-of the input features: y = X*w + c. The coefficient vector w is
-randomly sampled from a normal distribution, whereas the bias term c is
-set to a constant.
-
-As alpha tends toward zero the coefficients found by Ridge
-regression stabilize towards the randomly sampled vector w.
-For big alpha (strong regularisation) the coefficients
-are smaller (eventually converging at 0) leading to a
-simpler and biased solution.
-These dependencies can be observed on the left plot.
-
-The right plot shows the mean squared error between the
-coefficients found by the model and the chosen vector w.
-Less regularised models retrieve the exact
-coefficients (error is equal to 0), stronger regularised
-models increase the error.
-
-Please note that in this example the data is non-noisy, hence
-it is possible to extract the exact coefficients.
+When a model overfits, we want to regularize it's weights so it can generalize
+better on unseen data. Regularization reduces a model's reliance on specific
```

Comment:
I would slightly change the first sentence: I think that it could be nice to mention that a linear model that overfits will usually affect large weights to some features but the resulting association does not hold when it comes to unseen data. One way to fight overfitting is to constrain the weights value and it is called regularization.

Reply:
This looks good to me. I am only wondering about "even select fewer features" that would be linked to L1 regularization and Lasso models. It would be better to mention that it reduces the impact of features instead of "selecting".

Reply:
Yes, absolutely.